src/Workspaces/Remote/ServiceHub.CoreComponents/Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.csproj

```diff
@@ -18,31 +18,32 @@
     <!-- These references need to be deployed to the vsix subfolder containing servicehub bits for .Net Core -->
     <PackageReference Include="Microsoft.VisualStudio.Telemetry" Version="$(MicrosoftVisualStudioTelemetryVersion)" />
     <PackageReference Include="SQLitePCLRaw.bundle_green" Version="$(SQLitePCLRawbundle_greenVersion)" />
+    <!-- We use the content of these two runtime pacakges to exclude BCL library from the dependencies we need to deploy as part of Roslyn vsix. -->
+    <PackageReference Include="Microsoft.NETCore.App.Runtime.win-x64" Version="$(MicrosoftNetCoreAppPackagesVersion)" ExcludeAssets="all" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.WindowsDesktop.App.Runtime.win-x64" Version="$(MicrosoftNetCoreAppPackagesVersion)" ExcludeAssets="all" PrivateAssets="all" />
   </ItemGroup>
 
-  <Target Name="PublishProjectOutputGroup" DependsOnTargets="Publish" Returns="@(_PublishedFiles)">
+  <Target Name="PublishProjectOutputGroup" DependsOnTargets="Publish" Returns="@(_PublishedFiles)">  
+    <PropertyGroup>
+      <_RuntimeLibraryDir>$(NuGetPackageRoot)Microsoft.NETCore.App.Runtime.win-x64\$(MicrosoftNetCoreAppPackagesVersion)\runtimes\win-x64</_RuntimeLibraryDir>
```

Comment:
I think there is an msbuild property that contains path to package that is automatically defined by nuget.

Reply:
Yea, that's probably the biggest trouble I have working with msbuild - I have no clue what are available vs. what needs custom logic. Let me see if I can find this property you mentioned...